lib/python/picongpu/visualization/energy_histogram_visualizer.py

```diff
@@ -0,0 +1,120 @@
+from picongpu.plugins.energy_histogram import EnergyHistogram
```

Comment:
uh, forgot to mention: please add the copyright line to each file (see e.g. the data readers)